.gitignore

```diff
@@ -35,3 +35,9 @@ terraform.rc
 
 # IDEs
 .idea/
+
+#Certificates
+cerificate.*
+privatekey.pem
+certificate.pem
```

Comment:
Simplify the conditions, the last two are already covered. On the other hand, using the workdir for such files should be avoided